README.md

```diff
@@ -4,45 +4,68 @@ for emitting entities for the [Kythe](https://kythe.io) indexing schema.
 
 This is not an official Google product.
 
-# Instructions
+# Installation (Linux)
 
-## Prerequisites
+## Install Haskell compiler & tools (GHC & Stack).
```

Comment:
Why is the HP and GHC needed separately? Stack can install them (and won't use the system ones anyway).